p2p/conn/connection_test.go

```diff
@@ -1,7 +1,10 @@
 package conn
 
 import (
+	"bytes"
 	"encoding/hex"
+	"fmt"
```

Comment:
```suggestion
```
only accept this change if the below change is also accepted

Reply:
The `fmt` package is required for another `fmt.Sprintf` statement used to format the error message in the `assert.Ture`.